posts/2024-08-12_gc-us24/README.md

```diff
@@ -0,0 +1,103 @@
+---
+publication_date: 2024-08-12T00:00:00Z
+slug: gc-us24
+tags: []
+authors: [VT-Cosmos]
+---
+
+# gno.land at GopherCon US 2024
+
+
+GopherCon US is one of the largest events for the Go programming community.
+This year, in the vibrant city of Chicago, we had the honor of being the Diamond
+Sponsor at GopherCon US 2024. From July 7th to July 10th, we were surrounded by 
+top Go talent; it was an incredible opportunity for us to connect with developers,
+showcase our innovations, and share our expertise.
+
+GopherCon US 2024 was held at the stunning McCormick Place, bringing together
+nearly one thousand Go enthusiasts from around the world. Our team was thrilled
+to be part of this gathering, contributing to the vibrant ecosystem of Go
+developers.
+
+## Highlights of our collaboration
+
+### Jae’s Talk
+
+Jae Kwon, the founder of gno.land, gave a talk on “**Gno: Lessons in Building a
+Go Interpreter in Go.**”
+
+Jae’s  talk provided an in-depth overview of why this technology is gaining 
+traction in the development community. The discussion began with the key reasons
+to use Gno, and its seamless interoperability which allows for effortless 
+integration. Detailed comparisons were made between Gno and other prominent
+programming languages such as Solidity and Rust which are running other smart 
+contract platforms, showcasing Gno’s distinct advantages. The strengths of the
+GnoVM were a focal point, particularly its stack based AST architecture, its 
+auto persistence, as well as its determinism and other features. The presentation
+also shared valuable lessons learned from GnoVM's development and implementation,
+offering insights into best practices and challenges overcome in the future work. 
+Be sure to make some time to watch [the recording](https://youtu.be/betUkghf_jo)!
+
+### Gno workshop
+
+Participants had the opportunity to dive deep into the development process, 
+guided by our expert engineer **[Dylan Boltz](https://github.com/deelawn)** who hosted a workshop on Community 
+Day: **The Building a decentralized app on Gno.land**. The workshop provided practical 
+insights and step-by-step guidance, empowering attendees to start building their
+own applications. If you missed our workshop or want to revisit the session, you 
+can find a recorded version of the workshop [here](https://www.youtube.com/watch?v=lwL2VyjaV-A).
```

Comment:
This URL starts at 6:06

Reply:
Weird, starts normally for me. Can you check again, maybe in a incognito tab?

Reply:
My bad, my cookie was messing with me. Thanks for pointing it out, Leon!